rfi_file_monitor/applicationwindow.py

```diff
@@ -2,7 +2,7 @@
 gi.require_version("Gtk", "3.0")
 gi.require_version("Gdk", "3.0")
 from gi.repository import GLib, Gtk, Gdk
-from watchdog.events import FileSystemEventHandler, FileCreatedEvent, FileModifiedEvent
+from watchdog.events import FileSystemEventHandler, FileCreatedEvent, FileModifiedEvent, PatternMatchingEventHandler
```

Comment:
```suggestion
from watchdog.events import FileCreatedEvent, FileModifiedEvent, PatternMatchingEventHandler
```

This would fix the LGTM warning